data/weapons/weapon_data.csv

```diff
@@ -1,59 +1,59 @@
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-IBIS,diableavionics_ibis,0,,300,350,,25,,,90,4,24,10,8,FRAGMENTATION,3,,0,0.1,,,0,2,0.2,1,,1000,,,10,,,PD,"pd6, SR, diable_weapon_bp",,Diable Avionics,,Point Defense,,,,,,,,,,1.1
-Raptor Defense System,diableavionics_raptor,0,,1000,600,,40,,,75,8,48,10,4,FRAGMENTATION,10,,0,0.1,,,0,2,0.2,1,,1000,,,10,,,PD,"pd11, diable_weapon_bp",,Diable Avionics,,Point Defense,,,,,,,,,,
+IBIS,diableavionics_ibis,0,,300,350,,25,,,180,4,24,10,8,FRAGMENTATION,3,,0,0.05,,,0,2,0.2,1,,2000,,,10,,,PD,"pd6, SR, diable_weapon_bp",,Diable Avionics,,Point Defense,,,,,,,,,,1.1
+Raptor Defense System,diableavionics_raptor,0,,1000,600,,40,,,75,8,64,10,4,FRAGMENTATION,10,,0,0.05,,,0,2,0.2,1,,2000,,,10,,,PD,"pd11, diable_weapon_bp",,Diable Avionics,,Point Defense,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Recson S,diableavionics_recson,1,,700,800,,200,,,24,8,,,,KINETIC,225,,0,2,1,,2,6,5,6,,650,,,100,,,,"kinetic9, LR, diable_weapon_bp",,Diable Avionics,,Anti Shield,,,,,,,,,,
+Recson S,diableavionics_recson,1,,700,800,,250,,,24,8,,,,KINETIC,275,,0,1.8,1,,2,6,5,6,,650,,,100,,,,"kinetic9, LR, diable_weapon_bp",,Diable Avionics,,Anti Shield,,,,,,,,,,
 Recson V,diableavionics_recson2,2,0.75,2400,1000,,200,,,8,12,,,,KINETIC,250,,0,2.18,2,0.1,2,6,5,6,,650,,,100,,,,"kinetic14, LR, diable_weapon_bp",,Diable Avionics,,Anti Shield,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Artassault Revolver Cannon,diableavionics_artassault,0,,400,500,,75,,,30,5,,,,HIGH_EXPLOSIVE,75,,0,1.74,5,0.1,1,3,.25,1,,800,,,25,,,,"he8, diable_weapon_bp",,Diable Avionics,,Anti Small Craft,,,,,,,,,,
+Artassault Revolver Cannon,diableavionics_artassault,0,,400,500,,75,,,30,5,,,,HIGH_EXPLOSIVE,60,,0,1.74,5,0.1,1,3,.25,1,,800,,,25,,,,"he8, diable_weapon_bp",,Diable Avionics,,Anti Small Craft,,,,,,,,,,
 Artdeux Twin Cannon,diableavionics_artdeux,1,,1200,900,,40,,,12,11,30,7.5,15,HIGH_EXPLOSIVE,40,,0,0.05,2,0.05,1,6,0.1,2,,800,,,25,,,,"he12, diable_weapon_bp",,Diable Avionics,,Anti Hull,,,,,"%s: each projectile has chance to detonate dealing an additional %s damage, scales with the flux level of the firing ship.",Super-heated projectiles|80 Fragmentation,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Grave LAC,diableavionics_graveac,0,,600,500,,50,,10,30,6,9,1.666,3,KINETIC,50,,0,0.4,,,5,10,0.5,3,,900,,,100,,,,"kinetic8, SR, diable_weapon_bp",,Diable Avionics,,Anti Shield,,,,,,,,,,
+Grave LAC,diableavionics_graveac,0,,600,500,,70,,10,30,6,9,1.666,3,KINETIC,63,,0,0.4,,,5,10,0.5,3,,900,,,100,,,,"kinetic8, SR, diable_weapon_bp",,Diable Avionics,,Anti Shield,,,,,,,,,,
```

Comment:
Grave LAC damage increased to 70 (from 50), flux per shot increased to 63 (from 50)